### DISCUSSION TITLE
MONAI is FDA Approved + Full Example!

### DISCUSSION THREAD

**Discussion by yujanshrestha** [Project-MONAI/MONAI#7804] [General]
My team and I got a MONAI powered algorithm built and FDA approved in 12 months. 

I am a strong believer in sharing knowledge to help others. I managed to convince the company to allow the community to purchase the **documents we submitted to FDA.** 

This would have helped me immensely if I had such an example the first time I prepped a 510k!

I wrote an article about some best practices and pitfalls I came across in our FDA journey. I think it will be useful to the MONAI community.

Here is the article with examples: [510k Preparation Pitfalls: Best Practices, FAQs, and Examples](https://innolitics.com/articles/k-preparation-pitfalls-best-practices-faqs-and-examples/)

Here is the full example: [DIY 510(k) Accelerator](https://innolitics.com/services/diy-510k-accelerator/)

Are you an early stage cash-strapped startup? Regulatory is probably the biggest roadblock to get your algorithm into the hands of patients and providers. Let's change that. Your algorithm needs to get out there and save lives. I would love to support you. Let's work something out. [Email me](mailto:yshrestha@innolitics.com?subject=MONAI%20Startup%20Program) or hit me up on [LinkedIn](https://www.linkedin.com/in/yujanshrestha/).

![image](https://github.com/Project-MONAI/MONAI/assets/2153084/c30687c1-929f-42ab-bcfc-18727d69fdab)

**Comment by tangy5** [comment]
This is super amazing and inspiring! 
What a big milestone!

**Reply by yujanshrestha** [reply]
Thank you @tangy5!
